examples/example-workload/gatewayapi/blue-green/deployments.yaml

```diff
@@ -0,0 +1,125 @@
+apiVersion: apps/v1
```

Comment:
since this file contains both the workloads/services, *and* the HTTPRoute, I'd probably just call the file `blue-green.yaml`. Otherwise, could split the route into its own file if you wanted, but I'm not sure that's necessary.

Reply:
done